gapii/cc/call_observer.h

```diff
@@ -182,6 +182,12 @@ class CallObserver : public context_t {
   // observeTimestamp encodes a timestamp extra in the trace
   void observeTimestamp();
 
+  // setEndOfFrame marks this call as being an end of frame.
+  void setEndOfFrame() { mEndOfFrame = true; };
+
+  // getEndOfFrame returns true if this call is marked as an end of frame.
+  bool getEndOfFrame() { return mEndOfFrame; };
```

Comment:
Please remove the unnecessary semicolons at the end.

Reply:
Done, thanks!